lib/kbsecret/cli/kbsecret-cp

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+require "kbsecret"
+include KBSecret
+
+cmd = CLI.create do |c|
+  c.slop do |o|
+    o.banner = <<~HELP
+      Usage:
+        kbsecret cp <src-session> <target-session> [options] <record [record ...]>
+    HELP
+    o.bool "-m", "--move", "delete the record after copying"
+  end
+
+  c.dreck do
+    string :session
+    string :target_session
+    list :string, :labels
+  end
+
+  c.ensure_session!(:args)
+end
+
+target_session = Session.new(label: cmd.args[:target_session])
+cmd.args[:labels].each do |label|
+  cmd.die "No such record: #{label}" unless cmd.session.record?(label)
```

Comment:
This can be made a little more idiomatic/efficient:

```ruby
selected_records = src_sess.records.select { |r| cmd.args[:labels].include? r.label }
cmd.die "No such record(s)." if selected_records.empty?

selected_records.each do |record|
  dst_sess.import_record record
  src_sess.delete_record record if cmd.opts.move?
end
```

Reply:
Do you want the command to fail if one of the labels is missing?

Reply:
Also I'm wondering if it would be a good idea to catch any exceptions during import to prevent a record from being deleted if the import fails.

Reply:
Yes, the command should fail as early as possible (user asks to move a nonexistent record, user asks for nonexistent source and/or destination sessions, etc).

> Also I'm wondering if it would be a good idea to catch any exceptions during import to prevent a record from being deleted if the import fails.

`import_record` itself should throw an exception, killing the program. So long as we always `import_record` before we `delete_record`, that'll be sufficient.